internal/observability/event/eventer.go

```diff
@@ -558,7 +586,7 @@ func (e *Eventer) RotateAuditWrapper(ctx context.Context, newWrapper wrapping.Wr
 }
 
 // writeObservation writes/sends an Observation event.
-func (e *Eventer) writeObservation(ctx context.Context, event *observation) error {
+func (e *Eventer) writeObservation(ctx context.Context, event *observation, opt ...Option) error {
```

Comment:
you could use `_` instead of `opt` so it's clear that currenlty no options are supported.

Reply:
Done